src/VAC/VectorAnimationComplex/KeyFace.h

```diff
@@ -113,6 +125,9 @@ class Q_VPAINT_EXPORT KeyFace: public KeyCell, public FaceCell
     void triangulate_(Time time, Triangles & out) const;
 
     InfillPattern infillPattern_{};
+    double infillColor_[4] = {0, 0, 0, 0};
+    double infillHighlightedColor_[4] = {0, 0, 0, 0};
+    double infillSelectedColor_[4] = {0, 0, 0, 0};
```

Comment:
QColorF?

Reply:
In VPaint usualy is used the array double for store the colors, maybe because it is easy to pass to glColor4dv(const GLdouble*) function...
In the base classes also used double[] everywhere...

I can change this if you insist...

Reply:
No, I don't insist. 😅  Fine to keep it for now, and we'll consider something later on.

(Btw, glColor4fv() should allow you to send a float array just as easy. )